documentation/docs/strategies/cross-exchange-market-making.md

```diff
@@ -62,8 +75,9 @@ The following parameters are fields in Hummingbot configuration files (located i
+| **order_amount** | An amount expressed in base currency of maximum allowable order size.  If not set, the default value is 1/6 of the aggregate value of quote and base currency balances across the maker and taker exchanges.<br/><br/>*Example: assuming an order amount of `1` and a token symbol of ETH/DAI, the maximum allowable order size is one that has a value of 1 ETH.*
+| **adjust_order_enabled** | `True` or `False`<br/>If enabled (parameter set to `True`), the strategy will place the order on top of the top bid and ask if its more profitable to place it there. If this is set to `False`, strategy will ignore the top of the maker order book for price calculations and only place the order based on taker price and min_profitability. Refer to Adjusting orders section above. _Default value: True_
 | **active_order_canceling** | `True` or `False`<br/>If enabled (parameter set to `True`), Hummingbot will cancel that become unprofitable based on the `min_profitability` threshold.  If this is set to `False`, Hummingbot will allow any outstanding orders to expire, unless `cancel_order_threshold` is reached.
 | **cancel_order_threshold** | An amount expressed in decimals (i.e. input of `0.01` corresponds to 1%), which can be 0 or negative.<br/>When active order canceling is set to `False`, if the profitability of an order falls below this threshold, Hummingbot will cancel an existing order and place a new one, if possible.  This allows the bot to cancel orders when paying gas to cancel (if applicable) is a better than incurring the potential loss of the trade.
 | **limit_order_min_expiration** | An amount in seconds, which is the minimum duration for any placed limit orders. _Default value: 130 seconds_.
-| **top_depth_tolerance** | An amount expressed in quote currency of maximum aggregate amount of orders at a better price than Hummingbot's order that are allowed before Hummingbot modifies its order.<br/><br/>*Example: assuming a top depth tolerance of `100` and a Hummingbot bid price of 20, if there exist an aggregate amount of orders of more than 100 at a better (higher) price than Hummingbot's bid, Hummingbot will cancel its order at 20 and re-evaluate the next opportunity to place a new bid.*
-| **trade_size_override** | An amount expressed in quote currency of maximum allowable order size.  If not set, the default value is 1/6 of the aggregate value of quote and base currency balances across the maker and taker exchanges.<br/><br/>*Example: assuming a trade size override of `100` and a token symbol of ETH/DAI, the maximum allowable order size is one that has a value of 100 DAI.*
+| **top_depth_tolerance** | An amount in base currency, which is the specifies how much deep you want to go to calculate the top bid and ask. This is used for getting the top bid and ask, ignoring dust orders on top of the order book. _Default value: 0_.
```

Comment:
this definition is unclear. why did we modify the original?

Reply:
this also needs an example

Reply:
It is clear. The previous definition does not make sense to me. I will add example.

Reply:
read the sentence again. youre missing words.

Reply:
Vignesh changed the semantics from quote currency to base currency. But agreed the new doc is harder to understand and has grammatical errors (e.g. "which is the specifies ..."). The old doc is easier to understand in my opinion. We should probably just reuse that but changing the "quote currency" to "base asset".